src/Type/Php/ArrayFillKeysFunctionReturnTypeExtension.php

```diff
@@ -39,7 +40,11 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 		foreach ($constantArrays as $constantArray) {
 			$arrayBuilder = ConstantArrayTypeBuilder::createEmpty();
 			foreach ($constantArray->getValueTypes() as $keyType) {
-				$arrayBuilder->setOffsetValueType($keyType, $valueType);
+				if (! $keyType->toString() instanceof ErrorType) {
```

Comment:
Not sure its equivalent, but maybe you actually should do `$keyType->isString()->yes()` instead

Reply:
But I want to cast the type to string. To handle a case for example an object with `__toString` method.